hw/ip/aes/dv/aes_model_dpi/aes_model_dpi.h

```diff
@@ -25,42 +25,53 @@ extern "C" {
  *
  * @param  impl_i    Select reference impl.: 0 = C model, 1 = OpenSSL/BoringSSL
  * @param  op_i      Operation: 0 = encrypt, 1 = decrypt
- * @param  mode_i    Cipher mode: 6'b00_0001 = ECB, 6'00_b0010 = CBC,
- *                                6'b00_0100 = CFB, 6'b00_1000 = OFB,
- *                                6'b01_0000 = CTR, 6'b10_0000 = NONE
+ * @param  mode_i    Cipher mode: 7'b000_0001 = ECB, 7'000_b0010 = CBC,
+ *                                7'b000_0100 = CFB, 7'b000_1000 = OFB,
+ *                                7'b001_0000 = CTR, 7'b010_0000 = GCM,
+ *                                7'b010_0000 = NONE
```

Comment:
NONE is now 6'b11_1111. I've also left a comment somewhere else about this.

Reply:
Done.